invenio_app_ils/ui/package.json

```diff
@@ -24,6 +24,7 @@
     "react-searchkit": "^0.4.0",
     "redux": "^4.0.1",
     "redux-thunk": "^2.3.0",
+    "semantic-ui-calendar-react": "^0.15.0",
```

Comment:
Did you btw investigate different calendars, as this is the first one we add we should probably decide one that we will use along the whole application. I saw that our friends in Indico they are using the [airbnb](https://github.com/airbnb/react-dates) one.

Reply:
@zzacharo I played a bit with AirBnb dates [component](http://airbnb.io/react-dates/?path=/story/daterangepicker-drp--default). Despite the fact it is very neat, its usability is focused in selecting short date ranges. Try to select a period of a year in the demo, above and you will see that there are plenty of clicks required, the wider the range the worst it gets.

Reply:
> You didn't like the tab eh! :)

Not yet